coverage/.resultset.json

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
It's not necessary to set up simplecov in an assessment but great work getting 100% coverage! If you're worried about time though, feel free to skip it next time